test/test-mda-models/test-data-delivery-spark-model/src/main/resources/pipelines/SparkJavaDataDeliveryPatterns.json

```diff
@@ -68,14 +68,6 @@
 				"channelName": "outboundChannel"
 			}
 		},
-		{
-			"name": "VoidInboundAndMessagingOutboundAsync",
-			"type": "asynchronous",
-			"outbound": {
-				"type": "messaging",
-				"channelName": "outboundChannel"
-			}
-		},
```

Comment:
I: Seems to be having some issue with async messaging outbound without inbound messaging.
added https://github.com/boozallen/aissemble/issues/430 to follow up.